tessera-jaxrs/transaction-jaxrs/src/test/java/com/quorum/tessera/q2t/MockPrivacyGroupManager.java

```diff
@@ -0,0 +1,44 @@
+package com.quorum.tessera.q2t;
+
+import com.quorum.tessera.enclave.PrivacyGroup;
+import com.quorum.tessera.encryption.PublicKey;
+import com.quorum.tessera.privacygroup.PrivacyGroupManager;
+
+import java.util.List;
+
+public class MockPrivacyGroupManager implements PrivacyGroupManager {
```

Comment:
non-blocking/question: why do we declare a Mock (seems more like a stub) instead of using Mockito to create this mock at runtime?

Reply:
this is due to how we use static create method to instantiate the managers using `ServiceLoaderUtil` i.e. `return ServiceLoaderUtil.load(PrivacyGroupManager.class)`. Using the new `Mockito.mockStatic()` would be an alternative. 
However, these patterns will change when we migrate to jigsaw

Reply:
👍